src/components/MovieCard.jsx

```diff
@@ -1 +1,21 @@
-// implement MovieCard component here
+import React from 'react';
+import Rating from './Rating';
+
+class MovieCard extends React.Component {
+  render() {
+    const { title, subtitle, storyline, imagePath, rating } = this.props.movie;
```

Comment:
Mano, o ESLint reclamou lessa linha dizendo pra vc desconstruir o objeto?? Eu tinha feito igual e ele reclama ainda pedindo pra eu desconstruir... (Must use destructuring props assignmenteslintreact/destructuring-assignment)

Reply:
Sim, é porque ele reclama quando usamos muitas _dot notations_, uma forma pra fazer sem ele reclamar é assim:
```suggestion
    const { movie } = this.props;
    const { title, subtitle, storyline, imagePath, rating } = movie;
```
Ou então usar só o primeiro e passar como _movie.title_...

Reply:
Entendi, boa!! Acabei deixando do jeito que tá por gastar menos linhas...